CefSharp.Example/DownloadHandler.cs

```diff
@@ -19,7 +30,10 @@ public void OnBeforeDownload(IBrowser browser, DownloadItem downloadItem, IBefor
 
         public void OnDownloadUpdated(IBrowser browser, DownloadItem downloadItem, IDownloadItemCallback callback)
         {
-            
+            if (this.OnDownloadUpdatedFired != null)
```

Comment:
For consistency `this` has typically only been used when referencing a private variable.